groups/rds/profiles/heritage-development-eu-west-2/vars

```diff
@@ -155,3 +155,16 @@ parameter_group_settings = [
       value = "AUTO"
     },
 ]
+
+# Ingress security group patterns
+rds_ingress_groups = {
+  abbyy = [
+    "sgr-fes-app*"
+  ]
+  fes = [
+    "sgr-ewf-fe-asg*",
```

Comment:
I don't think we need access to the FES db from any of these services/SGs as they are all related to WebFiling which doesn't talk to Front End Scanning:
"sgr-ewf-fe-asg*",
 "sgr-ewf-bep-asg*",
 "ewf-frontend-tuxedo*",

But think we will need access from the CHS application subnet, for the EFS service, as that writes to the FES db directly.

Reply:
Resolved in [e50446f](https://github.com/companieshouse/frontend-scanning-service-terraform/pull/12/commits/e50446ffb0aec6616252483e25f7804ce078603d) and [8e4e13b](https://github.com/companieshouse/frontend-scanning-service-terraform/pull/12/commits/8e4e13b161dbfb26f2bf5c8686ec2b74dcd93025)